chcemvediet/apps/accounts/models.py

```diff
@@ -41,6 +41,12 @@ class Profile(FormatMixin, OriginalValuesMixin, models.Model):
                 default anonymization.
                 """))
 
+    days_to_publish_inforequests = models.IntegerField(null=True, blank=True, default=None,
```

Comment:
@martinmacko47 Co tak keby sme pre field nastavili `default=DAYS_TO_PUBLISH_INFOREQUEST` a `null=False`. Tym padom by sa nam aj vyrazne zjednodusil kod.

Reply:
@viliambalaz Urcite nie `default=DAYS_TO_PUBLISH_INFOREQUEST`, pretoze defaultnu hodnotu `DAYS_TO_PUBLISH_INFOREQUEST` mozno budeme casom menit. To by sposobovalo zakazdym migraciu.

Reply:
@martinmacko47 Skusal som si zmenit hodnotu v konstante `DAYS_TO_PUBLISH_INFOREQUEST`, system nedetekoval zmenu v migracii a pri vytvarani novych user-ov, sa im nastavili vzdy ta hodnota ktora bola definovana v konstante.

Reply:
@viliambalaz Lebo si zle spravil tu migraciu. Nenechal si si ju vygenerovat, ale upravil si si ju rucne. Nemozes mat:

```
from chcemvediet.apps.inforequests.constants import DEFAULT_DAYS_TO_PUBLISH_INFOREQUEST

...

field=models.IntegerField(default=DEFAULT_DAYS_TO_PUBLISH_INFOREQUEST, ...)
```

Migracia nesmie importovat konstanty z kodu. Musi pouzit explicitne cislo priamo v migracii:

```
DEFAULT_DAYS_TO_PUBLISH_INFOREQUEST = 60

...

field=models.IntegerField(default=DEFAULT_DAYS_TO_PUBLISH_INFOREQUEST, ...)
```

Lebo inac, ked zmenis hodnotu importnutej konstanty, tak sa zmeni hodnota aj v migracii, ktora sa vsak uz na produkcii spustila.

Reply:
Zmaz celu tu migraciu a nechaj si si ju vygenerovat djangom. Pozri rozdiel. A potom otestuj zmenu konstatny a vygenerovanie migracie nanovo.